docs/digger/developer/jenkins-build.asciidoc

```diff
@@ -1,2 +1,86 @@
 [[build-application]]
-=== Building Applications
+== Building Applications
+
+=== Building iOS Applications
+
+To build iOS Application using Aerogear Digger take the following steps below.
+
+. Create iOS Credentials
++
+Create a zipfile (using the extention .developerprofile) with your dev certificate and private key (in p12 file format) and a provisioning profile.
```

Comment:
The `.developerprofile` file extension is optional, it will work as long as it is a zipfile.

I think we should also mention that it will not work with password protected zipfiles.